boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts

```diff
@@ -0,0 +1,312 @@
+		#phy-cells = <0>;
+	};
+
+	/* RM0455 - 23.6 External device address mapping */
+	sdram2: sdram@d0000000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		device_type = "memory";
+		reg = <0xd0000000 DT_SIZE_M(16)>; /* 128Mbit */
+		zephyr,memory-region = "SDRAM2";
+		zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM) )>;
+	};
+
+	aliases {
+		led0 = &green_led;
+		pwm-led0 = &green_pwm_led;
+		sw0 = &user_button;
+	};
+};
+
+&clk_lsi {
+	status = "okay";
+};
+
+&clk_hsi48 {
+	status = "okay";
+};
+
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(25)>; /* X1: 25MHz */
+	status = "okay";
+};
+
+&pll {
+	div-m = <5>;
+	mul-n = <192>;
+	div-p = <2>;
+	div-q = <15>;
+	div-r = <4>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(480)>;
+};
+
+&usart1 {
+	pinctrl-0 = <&usart1_tx_pb6 &usart1_rx_pb7>;
+	pinctrl-names = "default";
+	current-speed = <57600>;
+	status = "okay";
+	dmas =
+		<&dmamux1 2 44 STM32_DMA_PERIPH_TX>,
+		<&dmamux1 3 43 STM32_DMA_PERIPH_RX>;
+	dma-names = "tx", "rx";
+};
+
+&usart3 {
+	pinctrl-0 = <&usart3_tx_pb10 &usart3_rx_pb11>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&rtc {
+	clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00010000>,
+		 <&rcc STM32_SRC_LSI RTC_SEL(2)>;
+	status = "okay";
+};
+
+&timers1 {
+	st,prescaler = <10000>;
+	status = "okay";
+
+	pwm11: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim1_ch1_pa8>;
+		pinctrl-names = "default";
+	};
+};
+
+&mac {
+	status = "okay";
+	/* MII */
+	pinctrl-0 = <&eth_ref_clk_pa1
+		     &eth_mdio_pa2
+		     &eth_crs_dv_pa7
+		     &eth_rxd2_pb0
+		     &eth_rxd3_pb1
+		     &eth_mdc_pc1
+		     &eth_txd2_pc2
+		     &eth_tx_clk_pc3
+		     &eth_rxd0_pc4
+		     &eth_rxd1_pc5
+		     &eth_txd3_pe2
+		     &eth_tx_en_pg11
+		     &eth_txd1_pg12
+		     &eth_txd0_pg13
+		     &eth_rx_er_pi10>;
+	pinctrl-names = "default";
+};
+
+&rng {
+	status = "okay";
+};
+
+&quadspi {
+	pinctrl-names = "default";
+	pinctrl-0 = <
+		&quadspi_bk1_io0_pd11
+		&quadspi_bk1_io3_pf6
+		&quadspi_bk1_io2_pf7
+		&quadspi_bk1_io1_pf9
+		&quadspi_clk_pf10
+		&quadspi_bk1_ncs_pg6
+		&quadspi_bk2_io2_pg9
+		&quadspi_bk2_io3_pg14
+		&quadspi_bk2_io0_ph2
+		&quadspi_bk2_io1_ph3
+		>;
+	flash-id = <1>;
+	status = "okay";
+
+	mt25ql512ab1: qspi-nor-flash-1@90000000 {
+		compatible = "st,stm32-qspi-nor";
+		reg = <0x90000000 DT_SIZE_M(64)>; /* 512 Mbits */
+		qspi-max-frequency = <72000000>;
+		spi-bus-width = <4>;
+		status = "okay";
+
+		partitions {
+			   compatible = "fixed-partitions";
+			   #address-cells = <1>;
+			   #size-cells = <1>;
+
+			   storage_partition: partition@0 {
+			       reg = <0x0 DT_SIZE_M(64)>;
+			   };
+		};
+	};
+
+	mt25ql512ab2: qspi-nor-flash-2@90000000 {
+		compatible = "st,stm32-qspi-nor";
+		reg = <0x90000000 DT_SIZE_M(64)>; /* 512 Mbits */
+		qspi-max-frequency = <72000000>;
+		status = "okay";
+	};
+};
+
+&dma1 {
+	status = "okay";
+};
+
+&dmamux1 {
+	status = "okay";
+};
```

Comment:
why is these enabled?

Reply:
DMA is used by usart1 for async API

Reply:
Ok, I have removed it from the dts and added the overlay to the test.